docs/content/_navigation.json

```
@@ -484,45 +484,16 @@
     "path": "/guides",
     "children": [
       {
-        "title": "Upgrading to Software-Defined Assets",
-        "path": "/guides/dagster/enriching-with-software-defined-assets"
+        "title": "General",
+        "path": "/guides/general"
       },
       {
-        "title": "Transitioning Data Pipelines from Development to Production",
-        "path":"/guides/dagster/transitioning-data-pipelines-from-development-to-production"
+        "title": "Best practices",
+        "path": "/guides/best-practices"
       },
       {
-        "title": "Testing Against Production with Dagster Cloud Branch Deployments",
-        "path": "/guides/dagster/branch_deployments"
-      },
-      {
-        "title": "Versioning and Memoization (Experimental)",
-        "path": "/guides/dagster/memoization"
-      },
-      {
-        "title": "Software-Defined Assets with Pandas and PySpark",
-        "path": "/guides/dagster/software-defined-assets"
-      },
-      {
-        "title": "Run Attribution (Experimental)",
-        "path": "/guides/dagster/run-attribution"
-      },
-      {
-        "title": "Migrating to Graphs, Jobs, and Ops",
-        "path": "https://docs.dagster.io/0.15.7/guides/dagster/graph_job_op",
-        "isExternalLink": true
-      },
-      {
-        "title": "Re-execution",
-        "path": "/guides/dagster/re-execution"
-      },
-      {
-        "title": "Fully-Featured Example Project",
-        "path": "/guides/dagster/example_project"
-      },
-      {
-        "title": "Validating Data with Dagster Type Factories",
-        "path": "/guides/dagster/dagster_type_factories"
+        "title": "Experimental features",
+        "path": "/guides/experimental-features"
```

```suggestion
        "title": "Experimental features",
        "path": "/guides/experimental-features"
        "children": [
        	{
          		"title": ...,
          		"path": "/guides/dagster/memoization"
          	},
          	{
          		"title": ...,
          		"path": "/guides/dagster/run-attribution"
          	}
        ]
```

should we also include the subpages as `"children"` in each section here like the suggestion^? so the guides can show up in the left nav

---

Yes!